docs/verifier/v3-circuit.md

```diff
@@ -18,10 +18,27 @@ We want to introduce new circuits - СredentialAtomicQueryV3 and CredentialAtomi
 :::warning
 
     Circuits are in the beta version. Trusted setup will be performed in the next release.
-    Current Circuit id is `СredentialAtomicQueryV3-beta.0`
+    Newest version is СredentialAtomicQueryV3-beta.1
+
+  <details><summary>Changelog from beta.0 to beta.1</summary>
+ 
+
+    1. ValueArrSize input is introduced, which fixes behavoiur for IN / NIN operations. 
+    2. Exists operator support.
+    3. Noop operator support.
```

Comment:
I think we should describe the utility of this attribute. Something in the line:

Noop enables a verifier to request proof of the presence of a credential in the user's wallet without requiring any additional conditions to be met or revealing any specific details of the credential.

Reply:
added link bellow the summary box.